x/stake/client/rest/query.go

```diff
@@ -194,7 +140,7 @@ func delegatorTxsHandlerFn(cliCtx context.CLIContext, cdc *wire.Codec) http.Hand
 		node, err := cliCtx.GetNode()
 		if err != nil {
 			w.WriteHeader(http.StatusInternalServerError)
-			w.Write([]byte(fmt.Sprintf("Couldn't get current Node information. Error: %s", err.Error())))
```

Comment:
Why aren't we keeping the more informative message? Is it already contained in `cliCtx.GetNode()`?

Reply:
Will leave the original msg